youtube_dl/extractor/sverigesradio.py

```diff
@@ -0,0 +1,67 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import compat_str
+from ..utils import (
+    try_get,
+)
+
+
+class SverigesRadioBaseIE(InfoExtractor):
+    _GEO_COUNTRIES = ['SE']
+
+    def _extract_audio(self, audio_type, audio_id):
+        audiourl = self._download_json('https://sverigesradio.se/sida/playerajax/getaudiourl?type=' + audio_type + '&quality=high&format=iis&id=' + audio_id, audio_id)
+        metadata = self._download_json('https://sverigesradio.se/sida/playerajax/audiometadata?type=' + audio_type + '&id=' + audio_id, audio_id)
```

Comment:
Query to `query`.

Reply:
You mean use a query dict? have done some refactor now